framework/cdv-gradle-config-defaults.json

```
@@ -4,7 +4,7 @@
     "GRADLE_VERSION": "6.8.3",
     "BUILD_TOOLS_VERSION": "30.0.3",
     "AGP_VERSION": "4.2.1",
-    "KOTLIN_VERSION": "1.4.32",
+    "KOTLIN_VERSION": "1.5.10",
```

```suggestion
    "KOTLIN_VERSION": "1.5.20",
```

latest is 1.5.20, so probably better to use that one